client/src/components/Header/styles.module.scss

```diff
@@ -61,4 +61,4 @@
       color: $footergrey;
     }
   }
-}
+
```

Comment:
curly bracket definitely should be returned, otherwise your app won't be built

Reply:
also `.header` selector contains 2 definitions of `position` property, please remove one